pkg/trtl/config/config_test.go

```diff
@@ -5,18 +5,24 @@ import (
 	"testing"
 	"time"
 
+	"github.com/rs/zerolog"
 	"github.com/stretchr/testify/require"
 	"github.com/trisacrypto/directory/pkg/trtl/config"
 )
 
 var testEnv = map[string]string{
-	"TRTL_ENABLED":                 "true",
-	"TRTL_REPLICA_BIND_ADDR":       ":445",
-	"TRTL_REPLICA_PID":             "8",
-	"TRTL_REPLICA_NAME":            "mitchell",
-	"TRTL_REPLICA_REGION":          "us-east-1c",
-	"TRTL_REPLICA_GOSSIP_INTERVAL": "30m",
-	"TRTL_REPLICA_GOSSIP_SIGMA":    "3m",
+	"TRTL_MAINTENANCE":              "true",
+	"TRTL_BIND_ADDR":                ":445",
+	"TRTL_LOG_LEVEL":                "debug",
+	"TRTL_CONSOLE_LOG":              "true",
+	"TRTL_DATABASE_URL":             "leveldb:///fixtures/db",
+	"TRTL_DATABASE_REINDEX_ON_BOOT": "true",
+	"TRTL_REPLICA_ENABLED":          "true",
+	"TRTL_REPLICA_PID":              "8",
+	"TRTL_REPLICA_NAME":             "mitchell",
+	"TRTL_REPLICA_REGION":           "us-east-1c",
+	"TRTL_REPLICA_GOSSIP_INTERVAL":  "30m",
+	"TRTL_REPLICA_GOSSIP_SIGMA":     "3m",
```

Comment:
Generally speaking this is the definitive list of environment variables used for configuration.